added guides for using custom logo in app (#1433)
Altinn/altinn-studio-docs: content/app/development/ux/logo/_index.en.md

---
title: Logo

> description: How to configure use of organization logo

Reviewer: Kanskje endre `description` til "How to set and configure the logo of an application" e.l.?

> {{% /notice%}}

Reviewer: Blir "Setting **the** application logo ..." på linja over mer riktig?

> ## Use logo from organization

Reviewer: ```suggestion
## Use organization logo
```

> To use the organization logo you can specify that in `applicationmetadata.json`:

Reviewer: Er litt vanskelig å se hva forskjellen på linje 30-36 og det som nevnes i `Use custom logo`. Så kan være greit å kanskje endre teksten her og fjerne linje 26. 
```suggestion
To use the organization logo specified in `altinn-orgs.json`, you can write the following in `applicationmetadata.json`:
```

Muligens flytte linje 28-36 under `Use custom logo` og ha 2 underoverskrifter: "From external URL" og "From application storage".

Author: Veldig enig

```json

> // applicationmetadata.json

Reviewer: Hvis du vil så kan du bruke `code-title`-shortcode for å vise hvilken fil det gjelder:
```
{{< code-title >}}
App/config/applicationmetadata.json
{{< /code-title >}}
```

> // applicationmetadata.json

Reviewer: Hvis du vil så kan du bruke `code-title`-shortcode for å vise hvilken fil det er snakk om:
```
{{< code-title >}}
App/config/applicationmetadata.json
{{< /code-title >}}
```

}
```

## Use custom logo

In order to use a custom logo, the logo must first be uploaded. 

1. Create a folder named `wwwroot`. This should be under App, `App/wwwroot`.
2. Upload a logo following the [design guidelines](#design-guidelines-for-app-owner-logo) to the folder.

> Each of the `resource.json` must reference the url the logo is in. 

Reviewer: Kan være lurt å spesifisere hvorfor det er viktig å oppdatere hver av filene/hvilke filer det gjelder?
```suggestion
Remember that each text resource file (`resource.[language].json`/`resources.XX.json`) must be updated for changes to apply to all languages.
```

```json
// resource.nb.json
{
  "id": "appLogo.url",
  "value": "/{appID}/{logoName}.svg"
}
```

The `appID` here refers to the `id` property in `applicationmetadata.json`.

> Finally, the `applicationmetadata.json` file needs to update the source of which the logo is from.

Reviewer: ```suggestion
Finally, `source` in `applicationmetadata.json` needs to be set to `resource` in order to access the logo we uploaded.
```

Evt. hvis det å sette logo fra URL flyttes så må denne kanskje stå i starten av "Use custom logo" hvis det stemmer at `source` må endres når man bruker `"appLogo.url"`.

> ```json
> // applicationmetadata.json
> "logo": {
>     "displayAppOwnerNameInHeader": false,
>     "source": "resource", // from "org" to "resource"
>     "size": "medium"
>   },
> ```

Reviewer: Kan benytte highlighting av linjer istedenfor kommentar, om det er ønskelig:
```suggestion
```json {hl_lines=[4]}
// applicationmetadata.json
"logo": {
    "displayAppOwnerNameInHeader": false,
    "source": "resource",
    "size": "medium"
  },
```
```
